src/sentry/models/integrations/external_actor.py

```diff
@@ -44,7 +45,12 @@ class Meta:
         unique_together = (("organization", "provider", "external_name", "actor"),)
 
     def delete(self, **kwargs):
-        install = self.integration.get_installation(self.organization_id)
+        from sentry.services.hybrid_cloud.integration import integration_service
+
+        integration = integration_service.get_integration(integration_id=self.integration_id)
+        install = integration_service.get_installation(
+            integration=integration, organization_id=self.organization.id
+        )
```

Comment:
At some point in the future we'll likely want to get this down to one RPC call. But that can be a task for future us.